src/traces/sankey/render.js

```diff
@@ -150,13 +150,17 @@ function sankeyModel(layout, d, i) {
 }
 
 function linkModel(uniqueKeys, d, l) {
-
     var tc = tinycolor(l.color);
     var basicKey = l.source.label + '|' + l.target.label;
     var foundKey = uniqueKeys[basicKey];
     uniqueKeys[basicKey] = (foundKey === void(0) ? foundKey : 0) + 1;
     var key = basicKey + (foundKey === void(0) ? '' : '__' + foundKey);
 
+    // for event data
+    l.trace = d.trace;
+    l.curveNumber = d.trace.index;
+    l.pointNumber = 'TODO';
```

Comment:
Thanks @etpinard I missed the question on the road, now see it's done, very nice! 🙇